test/uk/gov/hmrc/transitmovements/services/MovementFactoryImplSpec.scala

```diff
@@ -84,16 +84,17 @@ class MovementFactoryImplSpec
   "createEmptyMovement" - {
     val sut = new MovementFactoryImpl(clock, random)
 
-    "will create a movement without a message, movementEORINumber and movementReferenceNumber" in forAll(
+    "will create a movement with message, movementEORINumber and movementReferenceNumber" in forAll(
```

Comment:
The description here indicates we have movementEORINumber and movementReferenceNumber, can we clarify that these won't be present

Reply:
yes, updated